packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -283,3 +284,139 @@ export const ONE_SECOND_MS = 1000;
+            case '0x7fc77b5c7614e1533320ea6ddc2eb61fa00a9714':
+            case '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7':
+                return 150e3;
+            case '0xa2b47e3d5c44877cca798226b7b8118f9bfb7a56':
+                return 750e3;
+            case '0x45f783cce6b7ff23b2ab2d70e416cdb7d6055f51':
+                return 850e3;
+            case '0x79a8c46dea5ada233abaffd40f3a0a2b1e5a4f27':
+                return 1e6;
+            case '0x52ea46506b9cc5ef470c5bf89f17dc28bb35d85c':
+                return 600e3;
+            default:
+                throw new Error(`Unrecognized Curve address: ${poolAddress}`);
+        }
+    },
+    [ERC20BridgeSource.MultiBridge]: () => 350e3,
+    [ERC20BridgeSource.UniswapV2]: (fillData?: FillData) => {
+        // TODO: Different base cost if to/from ETH.
+        let gas = 90e3;
+        const path = (fillData as UniswapV2FillData).tokenAddressPath;
+        if (path.length > 2) {
+            gas += (path.length - 2) * 60e3; // +60k for each hop.
+        }
+        return gas;
+    },
+    [ERC20BridgeSource.SushiSwap]: (fillData?: FillData) => {
+        // TODO: Different base cost if to/from ETH.
+        let gas = 95e3;
+        const path = (fillData as SushiSwapFillData).tokenAddressPath;
+        if (path.length > 2) {
+            gas += (path.length - 2) * 60e3; // +60k for each hop.
+        }
+        return gas;
+    },
+    [ERC20BridgeSource.Balancer]: () => 120e3,
+    [ERC20BridgeSource.Cream]: () => 300e3,
+    [ERC20BridgeSource.MStable]: () => 700e3,
+    [ERC20BridgeSource.Mooniswap]: () => 220e3,
+    [ERC20BridgeSource.Swerve]: () => 150e3,
+    [ERC20BridgeSource.Shell]: () => 300e3,
+    [ERC20BridgeSource.MultiHop]: (fillData?: FillData) => {
+        const firstHop = (fillData as MultiHopFillData).firstHopSource;
+        const secondHop = (fillData as MultiHopFillData).secondHopSource;
+        const firstHopGas = DEFAULT_GAS_SCHEDULE[firstHop.source]!(firstHop.fillData);
+        const secondHopGas = DEFAULT_GAS_SCHEDULE[secondHop.source]!(secondHop.fillData);
+        return new BigNumber(firstHopGas)
+            .plus(secondHopGas)
+            .plus(30e3)
+            .toNumber();
+    },
+    [ERC20BridgeSource.Dodo]: (fillData?: FillData) => {
+        const isSellBase = (fillData as DODOFillData).isSellBase;
+        // Sell base is cheaper as it is natively supported
+        // sell quote requires additional calculation and overhead
+        return isSellBase ? 440e3 : 540e3;
+    },
+};
+// tslint:enable:custom-no-magic-numbers
+
+export const DEFAULT_GET_MARKET_ORDERS_OPTS: GetMarketOrdersOpts = {
+    // tslint:disable-next-line: custom-no-magic-numbers
+    runLimit: 2 ** 15,
+    excludedSources: [],
+    excludedFeeSources: [],
+    includedSources: [],
+    bridgeSlippage: 0.005,
+    maxFallbackSlippage: 0.05,
+    numSamples: 13,
+    sampleDistributionBase: 1.05,
+    feeSchedule: DEFAULT_GAS_SCHEDULE,
```

Comment:
We could technically also embed `FEE_SCHEDULE` because we do have a `PROTOCOL_FEE_MULTIPLIER` in constants. Though maybe it just makes more sense to move that constant out into a SwapQuoter opt.